### ISSUE TITLE
Where is James ?

### ISSUE THREAD

**Issue by hasanalpzengin** [mock-server/mockserver#1865]
Hello guys, there are no activity in the repo since 1 year and I can not find James is active in any other place. Anyone knows anything about him ?

**Comment by vitalijr2** [comment]
He is active in his own repos. e. g. [jamesdbloom/actions-runner-controller](https://github.com/jamesdbloom/actions-runner-controller)

**Comment by DamianFekete** [comment]
> He is active in his own repos

I don't think that is the case. From what I can see the changes that appear under his name are done by automatic tools (like Snyk and the like)

**Comment by jamesdbloom** [comment]
The project is under active development with new maintainers. Significant work has been completed recently including dependency updates, systematic PR triage (130+ PRs resolved), comprehensive issue analysis, and multiple bug fixes. A new release is being prepared and will be published shortly.
